packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -296,6 +308,7 @@ export function makeRumPublicApi(startRumImpl: StartRum, recorderApi: RecorderAp
      */
     addFeatureFlagEvaluation: monitor((key: string, value: any) => {
       strategy.addFeatureFlagEvaluation(sanitize(key)!, sanitize(value))
+      addTelemetryUsage({ feature: 'add-feature-flag-evaluation' })
```

Comment:
**💭 thought:** ‏almost all usages of `addTelemetryUsage` is only providing a feature name. It could be shorter to change the signature to something like:
```ts
addTelemetryUsage("feature", { optional_context: true })
```

Reply:
when introducing the API, I spent some time trying something like this but did not managed to have the typings working.
Don't hesitate to have a shot at it, happy to tweak that in a following PR.